runtime/devnet/src/extensions.rs

```diff
@@ -292,6 +304,42 @@ where
 	}
 }
 
+fn read_trust_backed_assets_state<T, E>(
```

Comment:
These are getting specific, consider a module so they become easier to maintain as others pallets are added.